using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md

```diff
@@ -76,3 +76,83 @@ public  void onReceive(Context context, Intent intent) {
 }
 ```
 
+## Process The Google Play Install Referrer
+
+You can use this API to process the data you get from the Google Play Install Referrer APIs.
+
+**Tip**: This API is available only in Android, since version `1.1.0`
```

Comment:
sorry, I missed this...please reword as follows:
"**Tip**: This API is available starting in Android version `1.1.0`."